templates/focus-ios.xliff

```diff
@@ -118,7 +147,7 @@
         <note>Label on About screen</note>
       </trans-unit>
       <trans-unit id="About.screenTitle">
-        <source>About</source>
+        <source>About Firefox Focus</source>
         <note>Title for the About screen</note>
```

Comment:
This shouldn't appear as "Firefox Focus" (see previous exports about that too)

Reply:
(should be a variable instead of product name. Else this will show up as Focus in Klar app, for example. We don't want pp to be able to change this)

Reply:
Correct, this should probably be `About $@` where the product name is filled in at runtime. We do this in other places too.